scripts/jupyter_gadi.sh

```diff
@@ -72,7 +78,9 @@ while getopts "$optspec" optchar; do
 done
 
 # This gets evaluated on Gadi in the SSH script
-WORKDIR=\$TMPDIR/runjp
+
+# Temporal files
+WORKDIR=/scratch/$PROJECT/$USER/tmp/runjp
```

Comment:
TMPDIR should already be `/scratch/$PROJECT/$USER/tmp`. The variables haven't been escaped here, they will be evaluated on the local machine, not Gadi. Note $USER will be empty by default at this point.

Reply:
The reason to change the code is that many of us belong to multiple projects and if the user login project (For example; `$PROJECT=v45`) differs from the selected project on the flag -P when running `jupyter_gadi -l jm5970 -q express -n 1 -m 1gb -P x77`, qsub will only mount the specified project in the flag -P. Then, the system variable `$TMPDIR=/scratch/v45/jm5970/tmp/runjp`, and the code will crash as qsub will only mount the scratch space defined by the flag -P (`/scratch/x77/jm5970/`). 

After explicitly defining the project of the qsub as the path, this issue is solved, as that scratch space is always mounted when the qsub session starts.